pandas/core/ops.py

```diff
@@ -766,7 +766,13 @@ def wrapper(self, other, axis=None):
             # always return a full value series here
             res = _values_from_object(res)
 
-        res = pd.Series(res, index=self.index, name=self.name, dtype='bool')
```

Comment:
Could you explain what you mean? The way I understand this, wrapper always return a Series even if the input is a SparseSeries. I would like wrapper to be able to return a SubclassedSeries if the input is a SubclassedSeries, but the return type would still have to be a Series if the input is a SparseSeries.